html/pfappserver/root/authentication/source/type/Kickbox.tt

```diff
@@ -0,0 +1,2 @@
+[% form.field('api_key').render | none %]
+[% form.field('email_required').render | none %]
```

Comment:
Required so that the config is populated properly as it's a true config parameter for that source